src/Schema/Builder.php

```diff
@@ -64,6 +78,24 @@ public function getIndexListing($table)
         return $processor->processIndexListing($results);
     }
 
+    /**
+     * @param string $table
+     * @return string[]
+     */
+    public function getForeignKeys($table)
```

Comment:
Is this code even necessary? It does the same thing as the parent method.

Reply:
Yea you may be right, will remove override and test

Reply:
Seems like the parent method was added 3 days ago and is only available in laravel/framework `v10.37.0`.
Can you bump the composer.json dependency to `10.37.3`?